lib/std/priority_queue.zig

```diff
@@ -18,33 +18,26 @@ pub fn PriorityQueue(comptime T: type, comptime Context: type, comptime compareF
     return struct {
         const Self = @This();
 
-        items: []T,
-        cap: usize,
-        allocator: Allocator,
+        items: []T = undefined,
+        /// The capacity of the queue. This may be read directly, but must not
+        /// be modified directly.
+        capacity: usize = 0,
```

Comment:
I made this change to match multi_array_list.zig and because this is the only standard library container I've seen with a `capacity()` function. It's not that I think the idea of a `capacity()` function is bad, just that it's possibly nonstandard.
 
In general it seems like there's not a fixed standard for getting the current size of a container or for getting its capacity, and oftentimes you have to access members directly - would be nice to standardize this across containers down the line.

Reply:
No, it was definitely bad. What you did is much better